array.c

```diff
@@ -2918,7 +2886,7 @@ rb_ary_reverse_bang(VALUE ary)
  *  call-seq:
  *    array.reverse -> new_array
  *
- *  Returns a new \Array whose elements are in reverse order:
+ *  Returns a new \Array with the elements of +self+ reversed.
```

Comment:
I think the previous sentence was more accurate.  The new text sounds like it does: `map(&:reverse)`.  Maybe: `Returns a new \Array with the elements of +self+ in reverse order.`